packages/mgt-components/src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -564,7 +564,7 @@ export class MgtPersonCard extends MgtTemplatedComponent {
     let email: TemplateResult;
     if (getEmailFromGraphEntity(person)) {
       email = html`
-        <div class="icon" @click=${() => this.emailUser()} tabindex=0>
+        <div class="icon" @click=${() => this.emailUser()} tabindex=0 role="button">
```

Comment:
@Mnickii / @musale I added the role onto these clickable divs as they act like a button so should likely have the role set accordingly, do you see any problems with doing this for a11y?

Reply:
@gavinbarron Yes. You need to add an aria-label for them so that you don't get ac11y errors for buttons without discernable text or missing aria-labels. You can set them as email, video etc

Reply:
Ahhh, thank you for that catch!

Reply:
We need to fix the styling of the text in here:
![image](https://user-images.githubusercontent.com/7122716/180857048-e3219bd8-113b-4af2-b8b3-ab100b0af30b.png)

Reply:
Commit added that resolves the font issue on those elements